roles/teuthology_nameserver/tasks/dnssec-keygen.yml

```diff
@@ -0,0 +1,44 @@
+#Temporarily moved dnssec key generation from deploy-teuthology script to the ansible task due to HMAC-MD5 being depricated on Tumbleweed.
+- name: Generate new keys
+  shell: "/usr/sbin/dnssec-keygen -r /dev/urandom -a HMAC-MD5 -b  512 -n HOST '{{ zone_name }}'"
+  register:
+    dns_keygen
+  tags:
+    - dnssec_keygen
+
+- name: Get key secret
+  shell: "grep 'Key:' {{ dns_keygen.stdout }}.private | cut -d ' ' -f 2"
```

Comment:
Since the private file is yaml file I hope ansible able to parse the yaml.

Reply:
Yeah... there is a filter `from_yaml`

Reply:
Isn't it more efficient what way? I mean we would need to have an extra step and var to get the key and use `from_yaml` and then save it to `secret`

Reply:
What do you call efficient? grep+cut?

Reply:
Done.